notebooks/04_model_select_and_optimize/tuning_spark_als.ipynb

```diff
@@ -37,16 +37,16 @@
   },
```

Comment:
<p>we are not testing this notebook, but theoretically it can be easily tested since we are not using hyperdrive, should we add an integration test for this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/864/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>this gets tested in the unit test in test_notebooks_pyspark::test_spark_tuning right?</p>

Reply:
Yea currently it is in the pyspark notebook test.